features/step_definitions/opportunities_steps.rb

```diff
@@ -15,6 +15,12 @@
   @result_count = page.first(:css, ".search-summary-count").text.to_i
 end
 
+When "I go to $brief URL address" do |brief|
```

Comment:
> Argh, I know I was the one who suggested writing a functional test for this but I'm wondering how much value it is adding. Sorry!

For our assertions, we are checking we see the temp message and no link to apply. We are already testing these things using our unit tests. What's our approach/strategy with our functional tests, should we be replicating? As this isn't visiting multiple pages as part of a flow I'd be tempted to say no...

Very open to thoughts/ideas

This has been discussed now and code altered accordingly.